YANG/ccamp/otn-tunnel/ietf-otn-tunnel.tree

```diff
@@ -648,66 +591,48 @@ module: ietf-otn-tunnel
             /te:label-start/te:te-label/te:technology:
     +--:(otn)
        +--rw otn-label
-          +--rw (range-type)?
-             +--:(trib-port)
-             |  +--rw tpn?   otn-tpn
-             +--:(trib-slot)
-                +--rw ts?    otn-ts
+          +--rw tpn?   otn-tpn
+          +--rw ts?    otn-ts
   augment /te:te/te:tunnels/te:tunnel/te:secondary-paths
             /te:secondary-path/te:path-in-segment
             /te:label-restrictions/te:label-restriction/te:label-end
             /te:te-label/te:technology:
     +--:(otn)
        +--rw otn-label
-          +--rw (range-type)?
-             +--:(trib-port)
-             |  +--rw tpn?   otn-tpn
-             +--:(trib-slot)
-                +--rw ts?    otn-ts
+          +--rw tpn?   otn-tpn
+          +--rw ts?    otn-ts
   augment /te:te/te:tunnels/te:tunnel/te:secondary-paths
             /te:secondary-path/te:path-in-segment
             /te:label-restrictions/te:label-restriction/te:label-step
             /te:technology:
     +--:(otn)
        +--rw otn-label-step
-          +--rw (range-type)?
-             +--:(trib-port)
-             |  +--rw tpn?   otn-tpn
-             +--:(trib-slot)
-                +--rw ts?    otn-ts
+          +--rw tpn?   otn-tpn
```

Comment:
Did not remember since when the two label-range for OTN (ts, tpn) were combined into one. In that case, the "range-type" parameter defined l1-types is meaningless. Just wanted to clarify this point.

Reply:
@agva123 the choice has been removed because it is redundant with the when statement (there is no choice after the range-type is configured). The when statement is sufficient to that make tpn applicable only when the range-type is tpn and ts applicable only when the range-type is ts

Reply:
We can consider not using the (technology) choice for the same reason but we have not been able to do this change in the topology model so the question is whether we prefer to be aligned with the topology model or the guidelines not to have redundant choice+when conditions